dbt/README.md

```diff
@@ -523,18 +530,24 @@ tests](https://docs.getdbt.com/best-practices/writing-custom-generic-tests),
 rather than [singular
 tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests).
 
-There are two subtypes of data tests that we support:
-
-1. **QC tests** confirm our assumptions about iasWorld data and are run at
-   scheduled intervals to confirm that iasWorld data meets spec
-2. **Non-QC tests** confirm all other assumptions about data sources outside
-   of iasWorld, and are run in an ad hoc fashion depending on the needs of
-   the transformations that sit on top of the raw data
-
-**QC tests** can be run using the [`dbt test`
-command](https://docs.getdbt.com/reference/commands/test) and their output can be
-transformed for analysis using the [`transform_dbt_test_results`
-script](https://github.com/ccao-data/data-architecture/blob/master/dbt/scripts/transform_dbt_test_results.py).
+Currently our primary use of data tests is to check assumptions about iasWorld data.
+We refer to this set of tests as "iasWorld data tests", and we've built a system
+for running and interpreting them that we will explain in the sections to follow.
+Other types of data tests do exist, and we primarily run them via automated
+GitHub workflows during CI when models change. However, we anticipate that in
+the future we will likely build out similar infrastructure for running and
+interpreting non-iasWorld data tests to accompany the infrastructure we have
+built for iasWorld data tests.
```

Comment:
Woops, I just made a comment on #571 asking to add _when_ we run tests which tests and you've basically already done that here, so nevermind.